### PR TITLE
Fix bibliography

### PR DESCRIPTION
**Pull request by philipwindecker**
#### Reference to a related issue in the repository
NA

#### Add a description
The ASAM office has chosen a different citation style going forward.
To be compliant with this new requirements, this branch and PR have been created.

#### Mention a member
@pmai 

#### Check the checklist

- [x] I have performed a self-review of my own code/documentation.
- [ ] My documentation changes are related to another repository in the organization. Here is the link to the issue/repo.
- [x] My changes generate no new warnings during the documentation generation.
- [x] The existing travis ci which pushes the documentation to gh-pages passes with my changes.

### CODE REVIEW

File: content/general_docs/bibliography.bib [hunk 1]
```diff
@@ -1,75 +1,75 @@
 % Encoding: UTF-8
 
-@Book{fmi2.0,
-  publisher = {Modelica Association, Linköping, Sweden},
-  title = {FMI 2.0: Functional Mock-up Interface for Model Exchange and Co-Simulation},
-  year = {2014-07},
+@techreport{FMIFunctionalMock,
```
Comment (pmai) [inline content/general_docs/bibliography.bib:3]:
```suggestion
@misc{fmi2.0,
```

This breaks the citations, so should ideally be referenced as fmi2.0, especially when we need fmi3.0 as well going forward. Also seems to me that handling this as @misc, like all the other standards seems more conistent...

File: content/general_docs/bibliography.bib [hunk 2]
```diff
@@ -1,75 +1,75 @@
...
+  organization = {{Modelica Association}},
+  number       = {https://fmi-standard.org/assets/releases/FMI_for_ModelExchange_and_CoSimulation_v2.0.pdf[FMI 2.0^], __Functional Mock-up Interface for Model Exchange and Co-Simulation v2.0__},
```
Comment (pmai) [inline content/general_docs/bibliography.bib:5]: Not sure this is the best way to handle the title, should likely be 
```suggestion
  title        = {https://fmi-standard.org/assets/releases/FMI_for_ModelExchange_and_CoSimulation_v2.0.pdf[FMI 2.0^], __Functional Mock-up Interface for Model Exchange and Co-Simulation v2.0__},
```

**Review by pmai — CHANGES_REQUESTED** [review]
This seems a bit inconsistent, and would require changes to the keywords we use to cite, so to be avoided. I'm also not sure that dropping publisher/organization info in the references across the board is a good idea...

**Comment by florianwallerer** [comment]
Should be conform to ASAM Editorial Guide, therefore needs to be updated.
@yash-shah-asam can we have a talk?

https://internal.pages.asam.net/asam-editorial-guide/editorial-guide-generator/editorialguide-asciidoc/1.0.0/Editorial_guide/02_structure.html#_bibliography

**Comment by AMuetsch** [comment]
> Should be conform to ASAM Editorial Guide, therefore needs to be updated. @yash-shah-asam can we have a talk?
> 
> https://internal.pages.asam.net/asam-editorial-guide/editorial-guide-generator/editorialguide-asciidoc/1.0.0/Editorial_guide/02_structure.html#_bibliography

I just saw that OSI has defined guidelines for a bibliography: 4.3.11. Guidelines for creating literary references
These look very sensible to me and are way more detailed than the ASAM editorial guide 2.1.10 Bibliography. 
Maybe the OSI guidelines should be incorporated into the editorial guide‽

**Pull request merged**